plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -73,3 +73,64 @@ Compressor.Threshold="Threshold (dB)"
 Compressor.AttackTime="Attack (ms)"
 Compressor.ReleaseTime="Release (ms)"
 Compressor.OutputGain="Output Gain (dB)"
+Crosshatching="Crosshatching"
+Crosshatching.Offset="Offset"
+Crosshatching.Threshold1="Threshold 1"
+Crosshatching.Threshold2="Threshold 2"
+Crosshatching.Threshold3="Threshold 3"
+Crosshatching.Threshold4="Threshold 4"
+CrossStitching="Cross Stitching"
+CrossStitching.StitchGap="Stitch Gap"
+CrossStitching.NegativeBias="Negative Bias"
+CrossStitching.StitchColor="Stitch Color"
+CrossStitching.StitchOpacity="Stitch Opacity"
+CrossStitching.InvertStitching="Invert Pattern"
+CrossStitching.Grayscale="Grayscale"
+DrosteEffect="Droste Effect"
+DrosteEffect.NumberOfRepeats="No. of Repeats"
```

Comment:
In the full language string, you use the abbreviation "No." instead of the word "Number".  Was there a particular reason for that (UI/UX issues displaying the string)?  I couldn't find any real precedent in the existing code base, and the only example of either usage/phrasing I could find was in the [linux-jack plugin](https://github.com/jp9000/obs-studio/blob/master/plugins/linux-jack/data/locale/en-US.ini#L2).